### ISSUE TITLE
Add node-mime

### ISSUE THREAD

**Issue by stephenlacy** [feross/render-media#1]
Noticed that you are using a json file with 14 mime types listed, would you accept a PR for adding `node-mime` for all common file types?

Cheers

**Comment by feross** [comment]
The mimetypes are only used when creating a blob to be added to the page via an image tag or iframe. It's really not that important for the mime to be set correctly for other types.

Also, `node-mime` is a huge file and this package is meant to be used in the browser.

```
$ browserify -r mime | wc -c
   41461
$ browserify -r mime | gzip | wc -c
   12795
```

The whole webtorrent script is only 80kb gzipped, so it's not worth it. Maybe we could do the top 100 file types as a compromise.

**Comment by stephenlacy** [comment]
Quite true - just checked a smaller lib, [simple-mime](https://github.com/creationix/simple-mime)

```
$ browserify -r simple-mime | wc -c       
6143

$ browserify -r simple-mime | gzip | wc -c
1885

```

Still to large? We could just import the types list from that module, as they are not exporting it right now.

**Comment by feross** [comment]
I pasted the mime types from `simple-mime` into here and removed a bunch that were obscure. There are lots more of the common ones now :)

Released as 2.0.1
